scss/prototype/_display.scss

```diff
@@ -0,0 +1,50 @@
+// Foundation for Sites by ZURB
+// foundation.zurb.com
+// Licensed under MIT Open Source
+
+////
+/// @group prototype-display
+////
+
+/// Responsive breakpoints for display, defaulted to global-prototype-responsive-breakpoints.
+/// @type Boolean
+$prototype-display-responsive-breakpoints: $global-prototype-responsive-breakpoints !default;
+
+@mixin foundation-prototype-display {
+  @each $display in (none, inline, inline-block, block, table, table-cell) {
```

Comment:
for example if we don't use text-justify they are 215 octect who don't use if we use only the small, medium and large breakpoint: 
```
.text-justify {text-align: justify;}
@media print, screen and (min-width: 39em) {.medium-text-justify {text-align: justify;}}
@media print, screen and (min-width: 60em) {.large-text-justify {text-align: justify;}}
```